Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+[^gamestar_de_westerner2]: [The Westerner 2: Westernheld Fenimore Fillmore ist er-
+wachsen geworden. Im Gegenzug
+schrumpft die Spielzeit auf Babyformat.](https://download.gamestar.de/public/43400/43452/05_2009_096.pdf)
+
+***
+
+## Supported Platforms:
+
+* light: Any (no software protection is employed, although it appears to mainly be marketed for Windows games regardless.)
+
+* professional: Windows
+(Some professional games are Hybrid PC/Mac discs, but the Mac versions of these games do not appear to utilize any sort of software protection.)[^108150]
+
+* audio: Audio CD
+
+***
+
+## Software Known to Detect:
+
+* ProtectionID
+* BinaryObjectScanner
+
+***
+
+## Preservation Instructions:
+
+light: None
+
+professional: Any sort of basic copy of the disc image appears to work.
+
+*** 
+
+## Associated File Attributes:
+
+### File Directories:
+
+#### "System":
+Directory for dummy files used by most professional discs so far, as well as some light[^Kenny] discs.
+
+#### "ZDAT":
+Directory for dummy files used by some, but not all light discs, and one known professional[^Petterson] disc so far. One calls it zdata[^108150], at least one uses System instead. Further variance likely.
+
+### File Names:
+Note: file names are not case sensitive, and vary from disc to disc.
+#### "gov_*.x64":
+Dummy file associated with professional. Names include gov_io.x64, gov_05.x64, and mov_io.x64.
+
+#### "iofile.x64":
+Dummy file associated with professional. Seems to be directly referenced by optgraph.dll.
+
+#### "sound.x64":
+Dummy file associated with professional. Seemingly not present on at least one professional disc.
+
+#### "optgraph.dll":
+Dll file responsible for software protection on professional discs.
+
+***
+
+## Known Manufacturing Information:
+
+### Known Manufacturing Plants:
+
+* optimal media production
+
+### Known Ringcode Information:
+
+#### Mastering Code:
+
+* manufactured by optimal media production	A######-0[Disc Number]
+* OH! Datenservice GmbH   www.oh-datenservice.de	A######-0[Disc Number]
+* KIDDINX	A######-0[Disc Number]
+
+#### Mastering SID Code:
+
+* IFPI L571
+
+#### Toolstamp or Mastering Code:
+
+* {Unordered list of toolstamp or mastering codes known to be associated with the DRM.}
+
+#### Mould SID Code:
+
+*  IFPI 97xx
+
+***
+
+# Reference Material:
+
+## Media: 
+
+Add example image to template is right, how do i put these in here
+1. [SoftICE for Windows 95 running before booting the game.](SoftICE_Detected.png)
+2. ["SoftICE for Windows 95" being detected.](SoftICE_Detected.png)
+this is from a specific game but idk how i put that on there iether
+***
+
+## Additional Resources:
+
+### Official Resources:
+
+[^CopyX_Original_Website]
+[^CopyX_2009_Document]
+
+***
+
+### Footnotes:
```

Comment:
Please use the latest archives of the websites that aren't currently cited as an archive, in-case the contents of the links change or get taken down. Exceptions are if the website isn't archived well (in which case this should be noted), or if it's something we have a level of control over, such as BOS pages.

Reply:
Mostly fixed except gamestar.de. You gave me permission in discord not to wayback the discogs links.